### ISSUE TITLE
Is there any way to delete a image file during job?

### ISSUE THREAD

**Issue by abidKiller** [openvinotoolkit/cvat#2403]
while annotating image I found a image i don't want to annotate , how can I delete that image?

**Comment by azhavoro** [comment]
duplicate of #95

**Comment by bsekachev** [comment]
There aren't way to delete a single image from the task. Do not annotate it if you know

**Comment by s3298230** [comment]
> There aren't any ways to delete a single image from the task. Do not annotate it if you know that the image is extra or recreate the task. 

A curious question, If we have frame without annotation in the dataset, will it impact the training?

**Comment by bsekachev** [comment]
It depends on training code and specific case I believe

**Comment by ktobah** [comment]
I think the possibility to delete an image from a task is a needed feature because imagine a context where annotating high-quality images is required but while annotating you face low-quality images and you want to remove them. You might say don't label them, however, in this particular task not annotating an image means the particular object(s) doesn't exist in the image (and you need such images in your training set)! Which is not the intended meaning. 

Another suggestion is to add maybe an extra-label "delete", then after you export your annotations, you can process the file and remove those images and the "delete" category. But this involves extra unnecessary processing.

**Comment by chad-green** [comment]
Yes, being able to delete as well as add images to the annotation task is a much needed feature!

**Comment by chad-green** [comment]
Here's scenario where this is important: You've upload 1,000 images for annotation of Object A. You think you have 1,000 quality images with clear examples of Object A, but when you get to image 999, you notice that Object A in dozens of the images is actually not an Object A and it could easily confuse the network. You're using a Yolo dataset so you can't leave the annotations blank or it will be considered a hard negative and ruin training. Currently, you'd have to make note of all those images and delete them from the database one by one after training. It would be much better if they could be deleted during a review or in real-time during annotation. 

Another nice feature that would really help would be some simple photo editing tools, like masking confounding objects. When you're really short on relevant images, you don't want to just throw away perfectly good images that just have a small confounding object. Drawing a filled black box or being able to crop out a section of the image would be super helpful.

**Comment by Nicofisi** [comment]
That is very true, we need this :(

**Comment by blafasel42** [comment]
Deleting or Marking images for SKIP during export would be very much appreciated. Also a way to Mask parts of the image and propagate the mask through many subesquent frames would be very helpful. Especially to prevent overfitting for parking cars in a traffic situation...

**Comment by bsekachev** [comment]
The feature will be released very soon, please, follow #95
